frontend/src/pages/DiscoverPage/DiscoverPage.tsx

```diff
@@ -1,8 +1,49 @@
-import StreamerGrid from "@/components/StreamerGrid/StreamerGrid"
+import { useState } from "react";
+import StreamerGrid from "@/components/StreamerGrid/StreamerGrid";
+import StreamerCard from "@/components/StreamerCard/StreamerCard";
 
-export default function DiscoverPage(){
-  return(
-    <StreamerGrid />
-  )
+type StreamerProfile = {
```

Comment:
Same here, don't copy paste but rather export these to be reused in multiple places